backends/p4tools/p4tools.def

```diff
@@ -122,6 +131,30 @@ class StateVariable : Expression {
     dbprint { ref->dbprint(out); }
 }
 
+#emit
```

Comment:
Would this be better in something like `irutils.h` or maybe something like `ir-compare.h`?